### DISCUSSION TITLE
Creating a custom voice package

### DISCUSSION THREAD

**Discussion by mikehgentry** [osmandapp/OsmAnd#14156] [Q&A]
Hi,

As per #10501 I'm trying to create packages for beep profiles. I've read the document here a few times https://osmand.net/features/custom-package , but I think it assumes more knowledge than I have, and I don't understand enough about programming to read the source and figure out what I'm missing. I also tried exporting existing voices to clue me in, but sadly it did not. All I've managed to get out of adb is "java.lang.IllegalArgumentException: No items" errors.

What I'm trying to do is pretty darn simple: create a directory 'beep-minimal' in voice, and put a .js and two .ogg files there. If anyone could put me out of my misery with a minimal .json (and appropriate structure for the .osf) that does this it'd be appreciated :)

Thanks!

**Comment by vshcherb** [comment]
You probably could start with already working voice package and then slowly update files in the folder. In https://github.com/osmandapp/OsmAnd/issues/10501 there is osf with working beeps, or in case it doesn't work you can download english voice package and update files inside
